### ISSUE TITLE
Silent message

### ISSUE THREAD

**Issue by Buranek** [kontalk/androidclient#836]
I sometimes need to write message to contact who suddenly fall asleep. So he is sleeping but signals are on. I want to answer but without disturbing him. Please add an ability to write message without initiating signal, only showing Android notification.

**Comment by daniele-athome** [comment]
Although I'd prefer to implement something "do not disturb during sleep hours" mode (from the receiver's side), this feature might be nice to have - since no other app has it for a start :-)

**Comment by Buranek** [comment]
No, that's not what I mean. Silent hours are available for Android notifications in many firmwares. Silent message is a fresh useful idea to answer without disturbing to a contact who **suddenly** fall asleep.

**Comment by daniele-athome** [comment]
I know, I was indeed talking about the opposite feature.
Anyway, how do you detect if the receiver has suddenly fall asleep? Or you just decide that he/she just did and manually activate this silent message feature?

**Comment by TheLastProject** [comment]
I just want to say that I think this feature is a horrible idea. The last thing I want is for my contact to decide when I get notifications.

**Comment by Buranek** [comment]
@daniele-athome I know this contact personally.

**Comment by webratte** [comment]
I absolutly agree with @TheLastProject

**Comment by daniele-athome** [comment]
> @daniele-athome I know this contact personally. And sometimes this contact is too tired to press something and just falls asleep.

Ok, this is a somewhat questionable scenario :-)

> I just want to say that I think this feature is a horrible idea. The last thing I want is for my contact to decide when I get notifications.

Although I agree that this a completely unimportant feature, what hurt could it do?
